src/main/java/org/kiwiproject/registry/model/ServiceInstance.java

```diff
@@ -61,6 +61,7 @@ public static ServiceInstance fromServiceInfo(ServiceInfo serviceInfo) {
                 .commitRef(serviceInfo.getCommitRef())
                 .description(serviceInfo.getDescription())
                 .version(serviceInfo.getVersion())
+                .metadata(serviceInfo.getMetadata())
```

Comment:
Should we make it a copy?

Reply:
Yes probably...`Map.copyOf` makes an _unmodifiable_ copy. Is that what we want?